src/main/js/distribution/Index.js

```diff
@@ -198,7 +199,8 @@ class Index extends Component {
     }
 
     copyButtonClick(cell, row) {
-        return <IconTableCellFormatter handleButtonClicked={this.copyButtonClicked} currentRowSelected={row} buttonIconName="copy" buttonText="Copy" />;
+        return <IconTableCellFormatter id="distribution" handleButtonClicked={this.copyButtonClicked}
```

Comment:
I'm not familiar with `IconTableCellFormatter`, but if it's a repeated field, it should probably have a dynamic suffix so that there are no errors related to uniqueness.

Reply:
Same thing for `editButtonClick` above this.

Reply:
Confirmed with QA that that unique ids for the cells won't benefit them as of now.  The edit button and copy button formatters will have different ids from each other.  I will update that.